torch_ort_inference/tests/bert_for_sequence_classification.py

```diff
@@ -0,0 +1,227 @@
+    prediction_sampler = SequentialSampler(prediction_data)
+    prediction_dataloader = DataLoader(prediction_data, sampler=prediction_sampler, batch_size=args.batch_size)
+
+    return prediction_dataloader
+
+def main():
+    # 1. Basic setup
+    parser = argparse.ArgumentParser(description='PyTorch BERT Sequence Classification Example')
+    parser.add_argument('--pytorch-only', action='store_true', default=False,
+                        help='disables ONNX Runtime inference')
+    parser.add_argument('--batch-size', type=int, default=1, metavar='N',
+                        help='input batch size for inference (default: 1)')
+    parser.add_argument('--export-onnx-graphs', action='store_true', default=False,
+                        help='export ONNX graphs to current directory')
+    parser.add_argument('--model-path', type=str, default=None,
+                        help='Path to fine tuned model for prediction')
+    parser.add_argument('--input', type=str, default=None,
+                        help="Input sentence for prediction")
+    parser.add_argument('--input-file', type=str, default=None,
+                        help="Input file in .tsv format for prediction")
+    parser.add_argument('--provider', type=str, default="openvino",
+                        help="ONNX Runtime Execution Provider for inference")
+    parser.add_argument('--backend', type=str, 
+                        help="OpenVINO target device for Inference.")
+    parser.add_argument('--precision', type=str,
+                        help="OpenVINO target device precision for prediction")
+
+    args = parser.parse_args()
+
+    # parameters validation
+    if not args.pytorch_only:
+        if args.provider != "openvino":
+            raise Exception("Invalid Provider! Set openvino as provider")
+        else:
+            if (args.backend is not None) and (args.backend not in list(ov_backend_precisions.keys())):
+                raise Exception("Invalid backend. Valid values are:", list(ov_backend_precisions.keys()))
+
+            if (args.precision is not None) and (args.precision not in ov_backend_precisions[args.backend]):
+                raise Exception("Invalid precision for provided backend")
+
+            if not (args.backend and args.precision):
+                print("OpenVINOExecutionProvider is enabled with CPU and FP32 by default." +
+                    " Please specify backend and precision to override.\n")
+                args.backend = "CPU"
+                args.precision = "FP32"
+
+            if args.backend == "MYRIAD":
+                print("Execution on MYRIAD might be impacted due to large model size")
+
+    # 2. Dataloader
+    # Load input dataset/file for prediction
+    prediction_dataloader  = load_pred_dataset(args)
+
+    # 3. Load Model
+    # If model path is not provided, use the model
+    # Else use a pretrained model fine-tuned on CoLA dataset from huggingface model hub
```

Comment:
All the comments need to be revisited since they still have references to older version.